apps/server/src/connect/dataEndpoints.test.ts

```diff
@@ -139,82 +76,17 @@ describe('dataEndpoints', () => {
   })
 
   describe('identityDataHandler', () => {
-    describe('validation', () => {
-      it('responds with a failure if connection_id is missing', async () => {
-        const res = {
-          send: jest.fn(),
-          status: jest.fn()
-        } as unknown as Response
-
-        await identityDataHandler(
-          {
-            query: {
-              provider: Providers.MX,
-              user_id: 'testUserId'
-            }
-          } as IdentityRequest,
-          res
-        )
-
-        expect(res.status).toHaveBeenCalledWith(400)
-        expect(res.send).toHaveBeenCalledWith('"connection_id" is required')
-      })
-
-      it('responds with a failure if user_id is missing', async () => {
-        const res = {
-          send: jest.fn(),
-          status: jest.fn()
-        } as unknown as Response
-
-        await identityDataHandler(
-          {
-            query: {
-              connection_id: 'testConnectionId',
-              provider: Providers.MX
-            }
-          } as IdentityRequest,
-          res
-        )
-
-        expect(res.status).toHaveBeenCalledWith(400)
-        expect(res.send).toHaveBeenCalledWith('"user_id" is required')
-      })
-
-      it('responds with a failure if provider isnt valid', async () => {
```

Comment:
I think this test would be a good one to keep.  I'm still suspicious that the endpoint will throw a 500 when the provider isn't valid because it's not getting caught. But if you keep this test then we could know for sure.